cmd/gosky/admin.go

```diff
@@ -78,7 +82,7 @@ var checkUserCmd = &cli.Command{
 
 		adminKey := cctx.String("admin-password")
 		xrpcc.AdminToken = &adminKey
-		xrpcc.Host = id.PDSEndpoint()
+		xrpcc.Host = cctx.String("admin-endpoint")
```

Comment:
makes sense. at some point we should probably have a better name than "admin" but i'm not sure what that should be right now

Reply:
yeah, i always lean towards more verbose when i'm unsure here